closure/templates/test/greeter_idom_proto.soy

```diff
@@ -0,0 +1,26 @@
+// Copyright 2016 The Closure Rules Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+{namespace io.bazel.rules.closure.soy.greeter autoescape="strict"}
+
+
+/**
+ * Greets a person.
+ */
+{template .greet}
+  {@param person: io.bazel.rules.closure.soy.Person}
```

Comment:
My mind is blown. I can't believe this is actually happening. We can now have protocol buffers at all layers of the stack. Thank you so much @hochhaus and @mikesamuel for making this possible.